images/regen-sandbox/setup/bridge.sh

```diff
@@ -0,0 +1,103 @@
+source $(dirname $0)/utils.sh
+
+set -eo pipefail
+
+TX_FLAGS="--from $ADDR1 --yes --fees 5000uregen"
+
+echo "INFO: Creating Credit Class - C01"
+regen tx ecocredit create-class $ADDR1 C "Bridging Credit Class" --class-fee 20000000uregen $TX_FLAGS | log_response
+
```

Comment:
Getting an error, 
```
ERROR: (code 4) failed to execute message; message index: 0: polygon is not an authorized bridge source: unauthorized
```

I'm guessing we need to somehow initiate polygon in `allowed-bridge-chains`?

Reply:
yeah, thats governance controlled, either need scripts to pass a proposal or modify the genesis to include polygon from the get go

Reply:
@technicallyty  Ok, so I could add my own script? (or we could add it here) that did something like:

```
regen tx gov draft-proposal
regen tx gov submit-proposal
regen tx gov vote
```
I don't know the specific sequence, but just verifying I got the right idea.

Reply:
@mhagel you're correct in the following tx commands would resolve this. I think updating to hardcode it in genesis is easier, as we don't need to wait for a voting window to close.

Addressed in [d9c581f](https://github.com/regen-network/regen-ledger/pull/1550/commits/d9c581ff039674b06f6e810a72ee059f92ee066e).